File: Java/CreationalDesignPattern/src/main/java/com/ute/refactoring/abstractfactory/factories/GUIFactory.java
package com.ute.refactoring.abstractfactory.factories;

import com.ute.refactoring.abstractfactory.buttons.Button;
import com.ute.refactoring.abstractfactory.checkboxes.Checkbox;

public interface GUIFactory {
    Button createButton();
    Checkbox createCheckbox();
}

File: C#/CreationalDesignPattern/CreationalDesignPattern/Refactoring/AbstractFactory/Factories/GUIFactory.cs
using CreationalDesignPattern.Refactoring.AbstractFactory;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CreationalDesignPattern.Refactoring
{
    public interface GUIFactory
    {
        AbstractFactory.IButton CreateButton();
        Checkbox CreateCheckbox();
    }
}
